docs/content/how-to/aws/migrate-hosted-cluster.md

```diff
@@ -0,0 +1,990 @@
+
+while [ ${NODE_POOL_REPLICAS} != ${NODE_STATUS} ]
+do
+    echo "Waiting for Nodes to be Ready in the destination MGMT Cluster: ${MGMT2_CLUSTER_NAME}"
+    echo "Try: (${count}/${timeout})"
+    sleep 30
+    if [[ $count -eq timeout ]];then
+        echo "Timeout waiting for Nodes in the destination MGMT Cluster"
+        exit 1
+    fi
+    count=$((count+1))
+    NODE_STATUS=$(oc get nodes --kubeconfig=${HC_KUBECONFIG} | grep -v NotReady | grep -c "worker") || NODE_STATUS=0 
+done
+```
+
+</details>
+
+## Teardown
+
+In this section we will shutdown and delete the HostedCluster in the source Management Cluster.
+> **NOTE:** Ensure you have the source's cluster's Kubeconfig placed as is set in `MGMT_KUBECONFIG` (if you follow the final script) or `KUBECONFIG` variable if you are going step by step. `export KUBECONFIG=${MGMT_KUBECONFIG}` or `export KUBECONFIG=<Kubeconfig FilePath>`
+
+1. Scale The Deployments and StatefulSets
+
+<details>
+<summary>ScaleDown Pod relevant objects in the HC ControlPlane Namespace</summary>
+
+```bash
+# Just in case
+export KUBECONFIG=${MGMT_KUBECONFIG}
+
+# Scale down deployments
+oc scale deployment -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --replicas=0 --all 
+oc scale statefulset.apps -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --replicas=0 --all 
+sleep 15
+```
+
+</details>
+
+2. Delete the NodePool objects
+
+<details>
+<summary>Delete NodePools</summary>
+
+```bash
+NODEPOOLS=$(oc get nodepools -n ${HC_CLUSTER_NS} -o=jsonpath='{.items[?(@.spec.clusterName=="'${HC_CLUSTER_NAME}'")].metadata.name}')
+if [[ ! -z "${NODEPOOLS}" ]];then 
+    oc patch -n "${HC_CLUSTER_NS}" nodepool ${NODEPOOLS} --type=json --patch='[ { "op":"remove", "path": "/metadata/finalizers" }]'
+    oc delete np -n ${HC_CLUSTER_NS} ${NODEPOOLS}
+fi
+```
+
+</details>
+
+3. Delete the Machines and MachineSets
+
+<details>
+<summary>Delete Machines and MachineSets</summary>
+
+```bash
+# Machines
+for m in $(oc get machines -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} -o name); do
+    oc patch -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} ${m} --type=json --patch='[ { "op":"remove", "path": "/metadata/finalizers" }]' || true
+    oc delete -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} ${m} || true
+done
+
+oc delete machineset -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --all || true
+```
+
+</details>
+
+4. Delete Cluster object
+
+<details>
+<summary>Delete the Cluster</summary>
+
+```bash
+# Cluster
+C_NAME=$(oc get cluster -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} -o name)
+oc patch -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} ${C_NAME} --type=json --patch='[ { "op":"remove", "path": "/metadata/finalizers" }]'
+oc delete cluster.cluster.x-k8s.io -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --all
+```
+
+</details>
+
+5. Delete the AWS Machines (Kubernetes Objects)
+    > **NOTE:** Don't worry about the real AWS Machines, even if you delete this object, the ControlPlaneOperator is down and will not affect the cloud instances
```

Comment:
nit: let's specificy CAPI controllers are down. Even if CPO is not running but CAPI is then that'd be a problem.